packages/gatsby/cache-dir/root.js

```diff
@@ -14,7 +14,11 @@ import EnsureResources from "./ensure-resources"
 
 import { reportError, clearError } from "./error-overlay-handler"
 
-if (window.__webpack_hot_middleware_reporter__ !== undefined) {
+// TODO: Remove entire block when we make fast-refresh the default
```

Comment:
Add an inline comment here explaining how we have kept this logic but it is in a different file